Tuvan tweaks and bug fixes

Thinking-waffle opened the pull request.

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update TAN_l_english.yml

Harain committed:
fix duplicate loc

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update soviet ft regaring the annexation of Tuva

Thinking-waffle committed:
Update r56_parties_l_english.yml

Thinking-waffle committed:
Update TAN.txt

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update TAN.txt

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update TAN - Tannu Tuva.txt

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update ITA.txt

Thinking-waffle committed:
Update BBA_Italy.txt

Thinking-waffle committed:
Update PER.txt

Thinking-waffle committed:
Update formable_nation_decisions.txt

Thinking-waffle committed:
Update 00_names.txt

Thinking-waffle committed:
Update usa.txt

Thinking-waffle committed:
Update xp_conversions_decisions.txt

-reduced chance of those being picked and won't be picked below 150 PP so they won't prevent the raise of conscription

Thinking-waffle committed:
extra conditions for late game US wargoals

Should prevent the US picking wargoals for countries it is already at war with, or alternatively is their ally

Thinking-waffle committed:
adding George Deatherage for fascist USA

Thinking-waffle committed:
Update USA - USA.txt

Thinking-waffle committed:
Adding George van horn moseley for the usa

Thinking-waffle committed:
Update USA - USA.txt

Thinking-waffle committed:
Update NOA - North Ossetia-Alania.txt

Thinking-waffle committed:
Extra historical Ossetian characters

(btw even the base game puts full names directly in those files)

Thinking-waffle committed:
Update NOA.txt

Thinking-waffle committed:
Update czechoslovakia.txt

Thinking-waffle committed:
Update HUN.txt

Thinking-waffle committed:
improved the Ukrainian focus to press the habsburg claim after feedback.

Thinking-waffle committed:
added missing generic supersonic plane for BBA

Thinking-waffle committed:
adding AAT Swedish planes to BBA system

Thinking-waffle committed:
Update units_planes.asset

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update TAN_l_english.yml

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update croatia.txt

Thinking-waffle committed:
Update units_infantry.asset

Harain committed:
update units_planes.asset

moved the R56 FIN_medium_plane_entity to where they are in vanilla, added a few missing R56 comments
uncommented ALL missing vanilla entities as per waffle's instructions (supersonic_jet_plane_entity is throwing an error)
this is an attempt to fix the following error:
[21:47:30][1947.01.28.04][gfxairplanes.cpp:681]: Failed to load entity of internal airplane scene: Mothership Aircraft (Mothership Aircraft)

Thinking-waffle committed:
Removed duplicate infantry model for one of the RKs

marijn211 committed:
Merge branch 'Beta-Edits' into waffle_last_minute_february

marijn211 reviewed (dismissed):
[19:50:40][no_game_date][trigger.cpp:564]: Error: "Non assign trigger is not enclosed in {}: has_country_leader, near line: 2549" in file: "common/national_focus/tuva.txt" near line: 2552
[19:50:43][no_game_date][triggerimplementation.cpp:3949]: common/national_focus/tuva.txt:2549: has_country_leader: in has_country_leader, either character, id or name is mandatory to find the character.
[19:50:43][no_game_date][trigger.cpp:117]: Trigger failed to validate: common/national_focus/tuva.txt:2549: has_country_leader

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update tuva.txt

Thinking-waffle committed:
Update TAN_l_english.yml

Thinking-waffle committed:
Update TAN - Tannu Tuva.txt

Thinking-waffle committed:
Update BUL_l_english.yml

Thinking-waffle committed:
fixed some bad effect in the generic industy tree

Thinking-waffle committed:
Ends exile gov economic ideas on uncapitulation

previously it restored normal economic and trade tools only at the end of a peace conference, which was useful but not enough.

Thinking-waffle committed:
Update r56_on_actions.txt

Thinking-waffle committed:
removed double "

marijn211 committed:
Merge branch 'Beta-Edits' into waffle_last_minute_february

marijn211 commented on common/national_focus/usa.txt:
```diff
@@ -6596,8 +6605,12 @@ focus_tree = {
+
 				NOT = {
-					has_government = democratic
+					OR = {
```
Just a reminder that OR is unnecessary within NOT blocks

marijn211 commented on localisation/english/BUL_l_english.yml:
```diff
@@ -12,6 +12,11 @@ l_english:
 
+# BFTB tooltips (replaced to correct the state numbers)
+ BUL_assert_our_claims_historical_claims_tt:0 "Grants cores on any historically claimed state fully controlled by §Y[BUL.GetNameWithFlag]§!.\nHistorically claimed states: §Y[106.GetName]§!, §Y[803.GetName]§!, §Y[77.GetName]§!, §Y[184.GetName]§!, §Y[731.GetName]§!, §Y[955.GetName]§!."
+ BUL_historical_states_tt:0 "\nHistorical territorial claims: §Y[106.GetName]§!, §Y[803.GetName]§!, §Y[77.GetName]§!, §Y[184.GetName]§!, §Y[731.GetName]§!, §Y[955.GetName]§!."
```
Might wanna throw these in the replace folder

marijn211 reviewed (changes requested):
Passed the diff check, passed the in-game test, passed the error log test *except for the error below* and up-to-date with main.

Please fix this
`
[22:11:18][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "GER_fighter_equipment_0_entity", associated with GER small_plane_airframe_0
[22:11:18][no_game_date][equipment_graphic_database.cpp:93]: Unknown equipment type: supersonic_fighter_equipment_1
[22:11:18][no_game_date][equipment_graphic_database.cpp:59]: Entity referenced in equipment graphic database does not exist: "ROM_fighter_equipment_1_entity", associated with ROM cv_small_plane_airframe_1`
And optionally resolve my two comments above

marijn211 committed:
Fix one error log thingie

Thinking-waffle committed:
Update 00_plane_icons.txt

Thinking-waffle committed:
Update 00_plane_icons.txt

Thinking-waffle committed:
Update units_planes.asset

Thinking-waffle committed:
Update 03_plane_icons.txt

Thinking-waffle committed:
Update 01_bba_plane_icons.txt

Thinking-waffle committed:
Update 01_bba_plane_icons.txt

Thinking-waffle committed:
Update 00_plane_icons.txt

Thinking-waffle committed:
Update 03_plane_icons.txt

marijn211 committed:
Merge branch 'Beta-Edits' into waffle_last_minute_february

marijn211 commented:
Passed the diff check, passed the in-game test, passed the error log test and up-to-date with main.

marijn211 committed:
Move localisation override to replace folder
